js/script.js

```diff
@@ -0,0 +1,309 @@
+
+// for (var n = 1000, num = 0; n > 50; n = n / 2, num++); 
+// console.log(n, num); //num-количество интеграций
+
+//==========switch-конструкция==========//
+
+// let num = prompt('Seasons', '');
+// switch (+num) {
+//   case 1:
+//     alert('Winter');
+//     break;
+//   case 2:
+//     alert('Spring');
+//     break;
+//   case 3:
+//     alert('Summer');
+//     break;
+//   case 4:
+//     alert('Autumn');
+//     break;
+//   default:
+//     alert("Нет таких значений");
+// }
+
+// let day = prompt('Day of mounth', '');
+// let message = (day < 1) ? 'unknown date' :
+//   (day < 10) ? 'first decade' :
+//   (day < 20) ? 'second decade' :
+//   (day < 32) ? 'third decade' :
+//   'unknown date';
+//   alert(message);
+
+// let month = prompt('month', '');
+// if (month == 12 || month <= 2){
+//   alert('winter');
+// } else if(month >= 3 && month <=5){
+//   alert('spring');
+// } else if(month >=6 && month <=8){
+//   alert('summer');
+// } else if(month >=9 && month <=11){
+//   alert('autumn');
+// }
+
+// let str = 'abcde';
+// if (str[0] == 'a') {
+// alert('Да');
+// } else {
+// alert('Нет');
+// }
+
+// let str = '123';
+// let sum = Number(str[0]) + Number(str[1]) + Number(str[2]);
+// alert(sum);
+
+// let str = '123456';
+// let str1 = Number(str[0]) + Number(str[1]) + Number(str[2]);
+// let str2 = Number(str[3]) + Number(str[4]) + Number(str[5]);
+// if (str1 == str2) {
+// alert('Да');
+// } else {
+// alert('Нет');
+// }
+
+// let str = 'abcde';
+// alert(str[0]);
+// alert(str[1]);
+// alert(str[4]);
+
+// let a = prompt('day','');
+// alert(60 * 60 * 24 * +a +' seconds');
+
+// let num = 1;
+// num = num + 12;
+// num = num - 14;
+// num = num * 5;
+// num = num / 7;
+// num = num + 1;
+// num = num - 1;
+// alert(num);
+
+// let num = 1;
+// num += 12;
+// num -= 14;
+// num *= 5;
+// num /= 7;
+// num += 1;
+// num -= 1;
+// alert(num);
+
+// let str1 = 'Hello, ';
+// let str2 = 'World!';
+// alert(str1 + str2);
+
+// let name ='Evgeniy';
+// alert('Hello,' + name);
+
+// let num = prompt('Enter the num','');
+// alert(num*num);
+
+// let num = '12345';
+// alert(num[0] * num[1] * num[2] * num[3] * num[4]);
+
+//==========Funcion==========//
+
+
+// function min(a, b) {
+//   if (a > b) {
+//     return 1;
+//   } else if(a<b){
+//     return -1;
+//   } else(a=b)
+//   return 0;
+// }
+// console.log(min(3,3));
+
+
+// showWords = function() {
+//   let args = [...arguments];
+//   document.write(args.join(' '));
+// }
+// showWords("I've", "been", "learning", "JavaScript", "for", "a", "month");
+
+// //второй вариант, как лучше?
+// showWords = function() {
+//   let result = '';
+//   let n = arguments.length;
+//   for(let i=0; i<n; i++) {
+//     result += arguments[i] + " ";
+//   }
+//   document.write(result);
+// }
+// showWords("I've", "been", "learning", "JavaScript", "for", "a", "month");
+
+// complexDigit = function() {
+//   const args = [...arguments];
+//   document.write(args.join(''));
```

Comment:
```suggestion
//   document.write([...arguments].join(''));
```

а еще можно сразу вот так, без лишней переменной